packages/cascara/src/atoms/Button/Button.js

```diff
@@ -14,7 +14,7 @@ const propTypes = {
   /** Main content of the button */
   content: pt.string,
   /** Makes the button take the width of the parent container */
-  fluid: pt.bool,
+  fluid: pt.string,
```

Comment:
```suggestion
  fluid: pt.bool,
```
This made it in during testing and should go back